modVM/src/org/aion/vm/AbstractExecutor.java

```diff
@@ -217,4 +258,9 @@ protected void updateRepo(ITxExecSummary summary, ITransaction tx,
             LOGGER.debug("Transaction logs: {}", summary.getLogs());
         }
     }
+
+    protected void setExecutionResult(IExecutionResult result) {
```

Comment:
not used

Reply:
it's used by the FastVM in the `fastvm_test` branch, you might have been on the `precompiled` branch